### ISSUE TITLE
It have so many errors and it's unusable it misses a lot of verses

### ISSUE THREAD

**Issue by mikiyasET** [magna25/amharic-bible-json#1]
መጽሐፈ ኢያሱ ወልደ ነዌ 4፥23 and 24 are missing for example this is just an example there are more missing verses like this

**Comment by magna25** [comment]
yeah, I'm a bit tied up right now so won't be able to fix it but any contribution is welcome for fixing these issues.
